src/list.scss

```diff
@@ -101,6 +101,107 @@ $block: #{$fd-namespace}-list;
         background: var(--sapList_Background);
       }
     }
+
+    &--action {
+      display: -webkit-box;
+      display: flex;
+      size: 2.5rem;
+      padding: 0 0.25rem;
+      -webkit-box-pack: center;
+      justify-content: center;
+      cursor: pointer;
+      border: var(--sapPageFooter_BorderColor);
+      background: var(--sapPageFooter_Background);
+      height: 3rem;
+
+      @include fd-ellipsis();
+
+      &--compact {
+        size: 1.625rem;
```

Comment:
I didn't see this value in the specs. I believe you used the values for "High Contrast Delta Specification".

Reply:
Added as per the below specification.
<img width="564" alt="Screen Shot 2020-04-17 at 12 21 06 PM" src="https://user-images.githubusercontent.com/53487468/79540624-14cb0f80-80a6-11ea-872a-a1dc1465c3ee.png">